cime_config/buildnml

```diff
@@ -31,13 +32,18 @@ sys.path.append(_CIME_CONFIG_PATH)
 
 # Import CAM's configure structure:
 from cam_config import ConfigCAM
+# HistoryConfig allows translation from user_nl_cam into Fortran namelists
+from hist_config import HistoryConfig
 
 #Import CAM's ParamGen class:
 from atm_in_paramgen import AtmInParamGen
 
 # Open CIME case log:
 _LOGGER = logging.getLogger(__name__)
 
+# Beginning of valid user_nl configuration line
+_USER_NL_LINE = re.compile(r"^[a-z][a-z0-9_]*[ ]*=", re.IGNORECASE)
+
```

Comment:
Just did a quick check and I think we can delete this now as there's no references to it.

Reply:
removed!